packages/@aws-cdk/aws-eks/lib/k8s-resource-attribute.ts

```diff
@@ -0,0 +1,87 @@
+import { Construct, CustomResource, Token, Duration } from '@aws-cdk/core';
+import { Cluster } from './cluster';
+
+/**
+ * Properties for KubernetesResourceAttribute.
+ */
+export interface KubernetesResourceAttributeProps {
```

Comment:
I am not sure about the term "resource". My understanding is that in k8s they are called "objects".

Also, this does not precisely represents an attribute...

Should this be: `KubernetesObjectValue`?